.github/workflows/release-old.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
     - name: Build with Gradle
-      run: ./gradlew -Dtests.security.manager=false build
+      run: ./gradlew -Dskip.integtests=true -Djava.security.manager=allow -Dbuild.snapshot=false
```

Comment:
skipping integ tests is temporary until we can build some more (might be able to remove). allowing the security manager and changes to the plugin-security.policy removed securitymanager violations.

Reply:
TermStatQuery tests also pass now.